lib/src/us/kbase/fbatools/ModelObjectSelectionParams.java

```diff
@@ -78,6 +81,21 @@ public ModelObjectSelectionParams withSaveToShock(Long saveToShock) {
         return this;
     }
 
+    @JsonProperty("fulldb")
```

Comment:
What is this and where did it come from?

Reply:
@bio-boris remake too.